Sources/MapboxDirections/Extensions/CoreLocation.swift

```diff
@@ -1,5 +1,9 @@
 import Foundation
+#if canImport(CoreLocation)
 import CoreLocation
+#else
+import Turf
+#endif
```

Comment:
This PR takes advantage of the compatibility shims for Core Location types that Turf provides as of mapbox/turf-swift#30. mapbox/turf-swift#119 tracks making Turf’s Core Location compatibility shim consistent with Polyline’s so it’s less magical and more obvious how to choose a particular shim.